translations/base-it.yaml

```diff
@@ -4,49 +4,49 @@ steamPage:
         infinita.
     discordLinkShort: Official Discord
```

Comment:
Not translated? Should be `Ufficiale` or something else like in Steam description